scss/footer.scss

```diff
@@ -0,0 +1,24 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  padding: 30px;
+  position: relative;
+
+  .footer-logo {
+    position: relative;
+    margin-top: 20px;
```

Comment:
it's margin bottom what you want since you want to separate the logo from the copyright